config/crd/bases/performance.openshift.io_performanceprofiles.yaml

```diff
@@ -1,6 +1,11 @@
+
```

Comment:
is this move needed by the new tooling or just suggested?

Reply:
Needed in order to use controller-gen/operator-sdk to create it.
We can probably go around and find a way to not move it here, but is now  a standard and also readers will expect the crd to be here.

Reply:
ack, I don't really mind.